docs/index.adoc

```diff
@@ -61,8 +61,13 @@ How it works: annotate your class and picocli initializes it from the command li
 
 TIP: Picocli also provides a <<Programmatic API,programmatic API>>, separately from the annotations API.
 
-=== Example application
-The example below shows a short but fully functional picocli-based `checksum` command line application:
+=== Example application: Checksum
+`Checksum` is a short but fully functional picocli-based command line application which prints the checksum (MD5 by default) of a file to STDOUT.
```

Comment:
For the same reason, I think the original description was better: I don't want to emphasize too much what the example does.

Reply:
Reverted to original version.